tests/testthat/test-dparse.R

```diff
@@ -1,3 +1,11 @@
+test_that("Parsing works correctly", {
+  #TODO: Add more per Github review
+  expect_s3_class(dparse("n()"), "Normal")
```

Comment:
Does this just work for R6?

Reply:
It works, but I am now using `distr6::expect_R6_class`.